Packs/AutofocusFeed/Integrations/AutofocusFeed/AutofocusFeed.yml

```diff
@@ -0,0 +1,122 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: AutoFocus Feed
+  version: -1
+configuration:
+- display: Indicator feed
+  name: indicator_feeds
+  options:
+  - Daily Threat Feed
+  - URL Feed
+  required: true
+  type: 16
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: The AutoFocus API key
+  name: api_key
+  required: true
+  type: 4
+- display: The URL feed ID to fetch
+  name: output_feed_id
+  required: false
+  type: 0
+- display: The name of the feed ID to fetch
+  name: output_feed_name
+  required: false
+  type: 0
+- display: Bypass exclusion list
+  name: bypassExclusionList
+  defaultvalue: ""
+  type: 8
+  required: false
+  additionalinfo: When selected, the exclusion list will be ignored for this feed.
```

Comment:
update the additionalinfo field value to what was decided in the thread in dmst-content-tip channel

Reply:
done